starport/services/network/path.go

```diff
@@ -4,13 +4,52 @@ import (
 	"os"
 	"path/filepath"
 	"strconv"
+
+	"github.com/tendermint/starport/starport/pkg/gentx"
 )
 
 // ChainHomeRoot is the root dir for spn chain homes
-const ChainHomeRoot = "spn"
+const (
+	ChainHomeRoot = "spn"
+	gentxPath     = "config/gentx/gentx.json"
+	genesisPath   = "config/genesis.json"
```

Comment:
We should normally be able to call `b.GenesisPath()` to get the genesis path of the chain